src/xdocs/config_system_properties.xml

```diff
@@ -74,5 +74,31 @@
       </p>
     </section>
 
+    <section name="Property chaining support">
+      <p>
+        Checkstyle supports property expansion within property definitions, also
+        known as property chaining.  This feature allows you to define properties
+        using other properties.  For example:
+        <source>
+checkstyle.dir=/home/user/checkstyle
+config.dir=${checkstyle.dir}/configs
+checkstyle.suppressions.file=${config.dir}/suppressions.xml
```

Comment:
please add one more line with property that depends on two properties to show that expansion is not simple

Reply:
I kept the same number of lines and changed the example a bit to show this.